ui/src/views/Insight.vue

```diff
@@ -1,19 +1,50 @@
 <template>
-  <div>
-    <h2>Insight</h2>
-    <div class="row">
-      <div class="col">
-        <div class="row">
-          <div class="col">
-            <h3>Files</h3>
-            <RemoteFiles />
-          </div>
-        </div>
+  <h2>Insight</h2>
+  <!-- FIXME: Cannot use tabs as it
```

Comment:
I don't think you should use tabs here. Tabs in tabs are generally confusing, but I think we already agreed upon a solution :)

Reply:
Will fix after merge